contracts/cw20-dao-crowdfund/src/contract.rs

```diff
@@ -32,19 +32,14 @@ pub fn instantiate(
         .querier
         .query_wasm_smart(dao_addr.clone(), &cw3_dao::msg::QueryMsg::GetConfig {})?;
 
+    // Verify fee manager address.
+    let fee_manager_addr = deps.api.addr_validate(msg.fee_manager_address.as_str())?;
```

Comment:
Can we do a basic smoke test that this is indeed a fee manager contract here by making a query to get its config?

Reply:
YES